README.md

```diff
@@ -21,7 +21,7 @@ Documentation around Clay is being refined in cojunction with iteration on this
 
 Clone the repo and run the following commands:
 
-- `cp .env.sample .env` (This will create a file with the required env variables)
+- `cp app/.env.sample .env` (This will create a file with the required env variables)
```

Comment:
We should add the path to `.env` too 
```suggestion
- `cp app/.env.sample app/.env` (This will create a file with the required env variables)
```

Reply:
Good catch on your part too! My latest commit addresses this!